payjoin/src/receive/v2.rs

```diff
@@ -364,11 +364,11 @@ impl UncheckedProposal {
     /// Broadcasting the Original PSBT after some time in the failure case makes incurs sender cost and prevents probing.
     ///
     /// Call this after checking downstream.
-    pub fn check_can_broadcast(
+    pub fn check_broadcast_suitability(
         self,
         can_broadcast: impl Fn(&bitcoin::Transaction) -> Result<bool, Error>,
     ) -> Result<MaybeInputsOwned, Error> {
-        let inner = self.inner.check_can_broadcast(can_broadcast)?;
+        let inner = self.inner.check_broadcast_suitability(None, can_broadcast)?;
```

Comment:
This v2 UncheckedProposal should have the same interface for `self.inner.check_boradcast_suitablity` and accept `min_feerate`. Hardcoding`None` should raise a yellow flag.